crypto/core_namemap.c

```diff
@@ -482,6 +573,7 @@ void ossl_namemap_free(OSSL_NAMEMAP *namemap)
 
     lh_NAMENUM_ENTRY_doall(namemap->namenum, namenum_free);
     lh_NAMENUM_ENTRY_free(namemap->namenum);
+    sk_OPENSSL_STRING_free(namemap->namedesc);
```

Comment:
This needs to be sk_OPENSSL_STRING_pop_free()

Reply:
Oh, good point!